File: tuto1/catkin_ws/src/my_robot_tutorials/src/publisher.cpp
#include <ros/ros.h>
#include <std_msgs/String.h>

int main(int argc, char **argv){
	ros::init(argc, argv, "cpp_robot_transmitter");
	ros::NodeHandle nh;
	ROS_INFO("Node has been started");
	ros::Publisher pub = nh.advertise<std_msgs::String>("/robot_transmitter", 10);
	ros::Rate rate(3);
	while(ros::ok()){
		std_msgs::String msg;
		msg.data = "message fron the CPP puplisher";
		pub.publish(msg);
		rate.sleep();
	}
	ROS_INFO("Node has been stopped");
}

File: tuto1/catkin_ws/src/my_robot_tutorials/scripts/publisher.py
#!/usr/bin/env python2

import rospy
from std_msgs.msg import String

if __name__ == '__main__':
	rospy.init_node('robot_transmitter')
	pub = rospy.Publisher("/robot_transmitter", String, queue_size=10)

	rate = rospy.Rate(2)
	while not rospy.is_shutdown():
		msg = String()
		msg.data = "hello, I am the robot ..."
		pub.publish(msg)
		rate.sleep()

	rospy.loginfo("Node stopped.")
